File: solve_problem/codeup/1092/solving_problem_together.c
#include <stdio.h>

int main(void)
{
    int day1, day2, day3;
    int day = 1;

    scanf("%d %d %d", &day1, &day2, &day3);

    while(day%day1 != 0 || day%day2 != 0 || day%day3 != 0) {
        day++;
    }
    
    printf("%d\n", day);

    return 0;
}

File: solve_problem/codeup/1000/1092/solving_problem_together.py

day1, day2, day3 = input().split()
day1 = int(day1)
day2 = int(day2)
day3 = int(day3)
day = 1

while(day%day1 != 0 or day%day2 != 0 or day%day3 != 0):
    day += 1

print(day)